### PR TITLE
Week 11 Assignment With Redux Exercise 1 & 2, And Hosted App On Netlify

### PR DESCRIPTION
**Pull request by jimishgajjar**

### CODE REVIEW

File: Week-11/redux-exercise-1/src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { createStore } from "redux";
+
```
Comment (subhadip-pesto) [inline Week-11/redux-exercise-1/src/redux/store.js:2]: please use proper folder structure to manage your redux actions, reducers and store

File: Week-11/redux-exercise-2/src/redux/reduser.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { actions } from "./actions";
+
```
Comment (subhadip-pesto) [inline Week-11/redux-exercise-2/src/redux/reduser.js:2]: check for all the typos, its "reducers"

Also give proper file names.

**Review by subhadip-pesto — CHANGES_REQUESTED** [review]

File: Week-11/redux-exercise-1/src/redux/store.js (hunk 1 above)
Comment (jimishgajjar) [reply Week-11/redux-exercise-1/src/redux/store.js:3]: Solved

File: Week-11/redux-exercise-2/src/redux/reduser.js (hunk 2 above)
Comment (jimishgajjar) [reply Week-11/redux-exercise-2/src/redux/reduser.js]: Solved

**Pull request merged**